benchmarks/Tosa/CMakeLists.txt

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The IREE Authors
```

Comment:
Could you change the directory name to `TFLite` so that we have benchmark names more externally facing? (TOSA, like MHLO, is internal stage for converting those input models.) This also helps if in the future we change the input model to the original TFLite flatbuffer (and goes through the import stage during benchmarking artifact generation).